src/main/resources/public/js/components/forms/form.js

```diff
@@ -14,11 +14,14 @@ Form.propTypes = {
     onSubmit: PropTypes.func.isRequired
 };
 
+const Label = ({ children, required, ...rest }) => 
+    <label {...rest}>{children + (required ? ' *' : '')}</label>
```

Comment:
Shouldn't this rather use a separate `{}` for the “required asterisk”?
```jsx
    <label {...rest}>{children}{required ? ' *' : '')}</label>
```
which btw could also be written
```jsx
    <label {...rest}>{children}{required && ' *'}</label>
```
(as _falsy_ values are not rendered)